release-common.nix

```diff
@@ -64,10 +64,7 @@ rec {
         apis = ["s3" "transfer"];
         customMemoryManagement = false;
       }).overrideDerivation (args: {
-        patches = args.patches or [] ++ [ (fetchpatch {
-          url = https://github.com/edolstra/aws-sdk-cpp/commit/3e07e1f1aae41b4c8b340735ff9e8c735f0c063f.patch;
-          sha256 = "1pij0v449p166f9l29x7ppzk8j7g9k9mp15ilh5qxp29c7fnvxy2";
-        }) ];
+        patches = args.patches or [] ++ [ ./transfermanager-content-encoding.patch ];
```

Comment:
I think you need `(arg.patches or []) ++ ...` :sob:

Reply:
Apparently not! Seems to work as expected poking at it in `nix repl`:

```
nix-repl> a = { x = [ "bar" ]; }

nix-repl> a.x or [] ++ [ "foo" ]
[ "bar" "foo" ]

nix-repl> a = { y = [ "bar" ]; }

nix-repl> a.x or [] ++ [ "foo" ]
[ "foo" ]
```